packages/platform-express/interfaces/nest-express-application.interface.ts

```diff
@@ -73,6 +75,24 @@ export interface NestExpressApplication extends INestApplication {
    */
   useStaticAssets(path: string, options?: ServeStaticOptions): this;
 
+  /**
+   * Register Express body parsers on the fly. Will respect
+   * the application's `rawBody` option.
+   *
+   * @example
+   * const app = await NestFactory.create<NestExpressApplication>(
+   *   AppModule,
+   *   { bodyParser: false, rawBody: true }
```

Comment:
```suggestion
   *   { rawBody: true }
```

I think if we omit that it would be less confusing. By readin the example I thought that we must have to disable `bodyParser` in order to use `useBodyParser`